### PR TITLE
Story on shahrukh khan and his fan.

### PR DESCRIPTION
**Pull request by iamsaumya**

### CODE REVIEW

**Comment by iamsaumya** [comment]
Please review this, @casey-collab .

**Comment by iamsaumya** [comment]
Please review this, @casey-collab

**Comment by casey-collab** [comment]
Hey! Me! Review this @casey-collab

File: hindi/Shahrukh-Khan/shahrukh-khan-fanboi.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+Tum soke uthe...Charo taraf dekha toh tum apne ghar pe nahi ho
```
Comment (casey-collab) [inline hindi/Shahrukh-Khan/shahrukh-khan-fanboi.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Pull request closed without merging**